lib/modules/platform/bitbucket-server/index.ts

```diff
@@ -670,7 +670,13 @@ async function updatePRAndAddReviewers(
     }
 
     // TODO: can `reviewers` be undefined? (#22198)
-    const reviewersSet = new Set([...pr.reviewers!, ...reviewers]);
+    const reviewersSet = new Set(
+      await Promise.all(
```

Comment:
```suggestion
      await Promise.allSettled(
```
otherwise we've lingering requests when one request fails early